tests/integration/models/application.spec.coffee

```diff
@@ -76,6 +76,7 @@ describe 'Application Model', ->
 					balena.pine.get
 						resource: 'application'
 						options:
+							$filter: is_public: false
```

Comment:
Is this meant to be part of the PR @thgreasi ?

Reply:
Nope, let me split it, good catch. I just had to added for the tests to pass on staging and forgot to reset before pushing.

Reply:
Done